model/lifecycle.go

```diff
@@ -1201,7 +1204,7 @@ func createOneTask(id string, creationInfo TaskCreationInfo, buildVarTask BuildV
 		ParentPatchNumber:       creationInfo.Build.ParentPatchNumber,
 		Version:                 creationInfo.Version.Id,
 		Revision:                creationInfo.Version.Revision,
-		MustHaveResults:         utility.FromBoolPtr(creationInfo.Project.GetSpecForTask(buildVarTask.Name).MustHaveResults),
+		MustHaveResults:         utility.FromBoolPtr(creationInfo.Project.FindProjectTask(buildVarTask.Name).MustHaveResults),
```

Comment:
Is this also supposed to be made panic-safe by checking for a nil project task?

Reply:
checking!